tools.go

```diff
@@ -0,0 +1,34 @@
+// +build tools
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This package contains code generation utilities
+// This package imports things required by build scripts, to force `go mod` to see them as dependencies
```

Comment:
Do the k8s authors provide this file in a way that we can import it from one of their repos?  Seems like that'd be the best option if possible.

Reply:
Good question, I'll see if that can work. The mentions of it that I thought were all towards adding the file instead of importing, tho.

Reply:
Haven't found one, there are a few different posts where it's suggested to "just add this file".